pkg/cluster/k8sres.go

```diff
@@ -147,14 +148,16 @@ func fillResourceList(spec acidv1.ResourceDescription, defaults acidv1.ResourceD
 	return requests, nil
 }
 
-func generateSpiloJSONConfiguration(pg *acidv1.PostgresqlParam, patroni *acidv1.Patroni, pamRoleName string, logger *logrus.Entry) string {
+func generateSpiloJSONConfiguration(pg acidv1.PostgresqlParam, patroni acidv1.Patroni, pamRoleName string, logger *logrus.Entry) string {
```

Comment:
was there any particular reason to change the type of parameter passing ?

Reply:
passing by pointer communicates intent of changing it, it helps  when reading unfamiliar codebase.

Reply:
reverted this change, even though I still think it is safer to pass by value